### PR TITLE
boxes layout update

### PR DESCRIPTION
**Pull request by muzammalrahim**
## Checklist

- [ ] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added documentation to describe my feature.
- [ ] I have squashed my commits into logic units.
- [x ] I have described the changes in the commit messages.

### CODE REVIEW

File: weblate_web/static/style-rtl.css [hunk 1]
```diff
@@ -4032,71 +4047,81 @@ input.fullwidth {
   }
 }
 
-/* Your Weblate Team - About Us */
```
Comment (nijel) [inline weblate_web/static/style-rtl.css:4035]: You remove "Your Weblate Team" section styling here:

image

Comment (muzammalrahim) [reply weblate_web/static/style-rtl.css:4035]: Fixing it!

Comment (muzammalrahim) [reply weblate_web/static/style-rtl.css:4035]: Please check the recent commit.

**Comment by nijel** [comment]
There are some code formatting issues left. In case the pull request would allow edits (see https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests), the pre-commit.ci service would do the formatting for you...

**Comment by muzammalrahim** [comment]
> There are some code formatting issues left. In case the pull request would allow edits (see https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests), the pre-commit.ci service would do the formatting for you...

Okay let me check!

**Comment by muzammalrahim** [comment]
> There are some code formatting issues left. In case the pull request would allow edits (see https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests), the pre-commit.ci service would do the formatting for you...

I have pre commit installed already, it's passing the checks from my side but it fails on Git.

**Comment by nijel** [comment]
You can ignore the DeepSource: Python errors, it sometimes fails to properly detect changes in the source and complains on errors which were present in the repository before.

**Comment by muzammalrahim** [comment]
> You can ignore the DeepSource: Python errors, it sometimes fails to properly detect changes in the source and complains on errors which were present in the repository before.

Okay, other than that all other checks managed to pass. Should i create the PR for previous fixes now?

**Comment by nijel** [comment]
Merged, thanks for your contribution!

We can now focus on getting https://github.com/WeblateOrg/website/pull/486 merged as well.

**Comment by nijel** [comment]
Sorry, I meant https://github.com/WeblateOrg/website/pull/474...

**Pull request merged**